### ISSUE TITLE
affineAlgebra ignores CoefficientField option

### ISSUE THREAD

**Issue by foxjwill** [Macaulay2/M2#636]
_I've only tested this on M2v1.9, but I didn't find any notes about this issue in the issue tracker, so I'm assuming (correctly?) that it's still an issue in M2v1.10._

This is about the `affineAlgebra` method from the package `MonomialAlgebras`. According to the documentation, the code `affineAlgebra(B, CoefficientField=>QQ)` (where `B` is a list, e.g. `{{1,2},{3,4}}`) should return a ring with coefficient field `QQ`. Instead, it always returns a ring with coefficient field `ZZ/101`. 

Looking at the code, the problem is that `opt` is not being passed to `binomialIdeal`:
```
-- code for method: affineAlgebra(List)
[redacted]/macaulay2-1.9.2/share/Macaulay2/MonomialAlgebras.m2:134:26-136:10: --source code:
affineAlgebra List := opt-> B -> (
I:=binomialIdeal B;
(ring I)/I)
```

**Comment by DanGrayson** [comment]
Thanks, good point.  I'm referring it to David.

**Comment by d-torrance** [comment]
Closed by #3278
